adapter_pipelines/ss2_single_sample/adapter_example_static_demo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "AdapterSs2RsemSingleSample.gtf": "gs://hca-dcp-mint-test-data/reference/demo/gencodev19_chr21.gtf",
```

Comment:
This should be `AdapterSmartSeq2SingleCell` now since `AdapterSs2RsemSingleSample` gets deleted in this PR.